src/LuaTransformer.ts

```diff
@@ -4634,6 +4636,93 @@ export class LuaTransformer {
         return this.createSelfIdentifier(thisKeyword);
     }
 
+    public transformTaggedTemplateExpression(expression: ts.TaggedTemplateExpression): ExpressionVisitResult {
+        const strings: ts.StringLiteral[] = [];
+        const rawStrings: string[] = [];
+        const expressions: ts.Expression[] = [];
+
+        if (ts.isTemplateExpression(expression.template)) {
+            // Expressions are in the string.
+            strings.push(ts.createStringLiteral(expression.template.head.text));
+            // If there is a head, it beings with "`" and ends with "${"
+            rawStrings.push(
+                expression.template.head
```

Comment:
Maybe it's possible to simplify it a bit? TypeScript version doesn't do any direct replacements: https://github.com/microsoft/TypeScript/blob/37b20f6afd3eee694c26ca39c586c42ec97be98e/src/compiler/transformers/es2015.ts#L3996-L4014

Reply:
Thanks for the link, I've created `tsHelper.getRawLiteral` to neaten up the method some more